### DISCUSSION TITLE
409 Client error when starting containers

### DISCUSSION THREAD

**Discussion by mwigginsdev** [nextcloud/all-in-one#2090] [Questions] [answered]
I have a fresh AIO setup on a debian server running docker.

I got everything set up correctly I think, domain validation passes. I get through to the /containers page and when i hit the "start containers" button i get the following error:

```
#35 {main}
NOTICE: PHP message: Slim Application Error
Type: GuzzleHttp\Exception\ClientException
Code: 409
Message: Client error: `DELETE http://localhost/v1.41/containers/nextcloud-aio-collabora?v=true` resulted in a `409 Conflict` response:
{&quot;message&quot;:&quot;You cannot remove a running container c43498bbc9f0a0efa5a746a5825b4c89fb46e1ad373d8e472838a31e20f94edb. Stop (truncated...)
File: /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php
Line: 113
```

If i manually stop the docker container listed there `c4349...` it fails on a different container and if I keep going it loops through to the same docker containers with different ids.

Any thoughts/help? This may be something super simple that I'm missing

**Comment by szaimen** [comment, marked as answer]
Hi, this looks like a hickup. Can you please reset AIO properly and try again? See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance

**Reply by mwigginsdev** [reply]
That worked thanks!
